src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -1,57 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we consider having proto files all have same names but under a subfolder so it is only a path to change rather than putting suffix everywhere wherever it is used?

I'm not clear of the implication, but it feels it would make the .proj / .targets file simpler and overal build process easier to understand if that could be done.